Exercises/1-callback.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
-const iterate = (obj, callback) => null;
+const iterate = (obj, cb) => {
+  Object.keys(obj).forEach(key => {
```

Comment:
Use loops, forEach is not a good idea here.

Reply:
✅